test/LiquidDemocracy.test.js

```diff
@@ -7,20 +9,54 @@ const should = require('chai')
   .should();
 
 contract('LiquidDemocracy', function (accounts){
-    var proposals = ['yes', 'no'];
-    
-    describe('setting up liquid democracy', function () {
-      it('should create ballot instance with provided proposals', async function () {
-        this.liquidDemocracyBallot = await LiquidDemocracy.new(proposals);
-        should.exist(this.liquidDemocracyBallot);
-      });
-      
-      it('should register new voters', async function () {
-        this.liquidDemocracyBallot = await LiquidDemocracy.new(proposals);
-        await this.liquidDemocracyBallot.registerNewVoter({ from: accounts[0] });
-        const voter1 = await this.liquidDemocracyBallot.getVoterData(accounts[0]);
-        should.exist(voter1);
-      });
-      
+
+  beforeEach(async function () {
+    // Set up ballot instance and register 4 voters
+    this.liquidDemocracyBallot = await LiquidDemocracy.new();
+    for (let i = 0; i < 4; i++) {
+      await this.liquidDemocracyBallot.registerNewVoter({ from: accounts[i] });
+    }
+  });
+  
+  describe('registration set up', function () {
+    it('should register voters successfully', async function () {
+      const registered = await this.liquidDemocracyBallot.getVoterData(accounts[0]);
+      let registeredWeight = registered[0].toString();
+      let registeredStatus = registered[1];
+      const unregistered = await this.liquidDemocracyBallot.getVoterData(accounts[4]);
+      let unregisteredWeight = unregistered[0].toString();
+      let unregisteredStatus = unregistered[1];
+  
+      registeredWeight.should.equal('1');
+      registeredStatus.should.equal(true);
+      unregisteredWeight.should.equal('0');
+      unregisteredStatus.should.equal(false);
     });
+  });
+  
+  describe('voting', function () {
+    it('should allow registered voters to vote', async function () {
+      await this.liquidDemocracyBallot.vote(accounts[0], 0);
+      await this.liquidDemocracyBallot.vote(accounts[1], 0);
+      await this.liquidDemocracyBallot.vote(accounts[2], 1);
+      const voteCountOption0 = await this.liquidDemocracyBallot.getBallotVoteCount(0);
+      const voteCountOption1 = await this.liquidDemocracyBallot.getBallotVoteCount(1);
+
+      voteCountOption0.toNumber().should.equal(2);
```

Comment:
is this really needed? I think getBallotVoteCount should already return an int

Reply:
You mean the `toNumber()` pipe? Yea because [Truffle gives back a BigNumber](http://truffleframework.com/docs/getting_started/contracts#making-a-call) object for getters such as `getBallotVoteCount()`, so it's needed if we want set up test with the `should.equal()` method